ci/azure/macos_arm64_script

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
+
+set -x
+set -e
+
+brew install s3cmd ninja gnu-tar
+
+ZIGDIR="$(pwd)"
+ARCH="aarch64"
+# {product}-{os}{sdk_version}-{arch}-{llvm_version}-{cmake_build_type}
+CACHE_HOST_BASENAME="llvm-macos10.15-x86_64-11.0.1-release"
+CACHE_ARM64_BASENAME="llvm-macos11.0-arm64-11.0.1-release"
+PREFIX_HOST="$HOME/$CACHE_HOST_BASENAME"
+PREFIX_ARM64="$HOME/$CACHE_ARM64_BASENAME"
+JOBS="-j2"
+
+rm -rf $PREFIX
+cd $HOME
+wget -nv "https://ziglang.org/deps/$CACHE_HOST_BASENAME.tar.xz"
+wget -nv "https://ziglang.org/deps/$CACHE_ARM64_BASENAME.tar.xz"
+
+gtar xf "$CACHE_HOST_BASENAME.tar.xz"
+gtar xf "$CACHE_ARM64_BASENAME.tar.xz"
+
+cd $ZIGDIR
+
+# Make the `zig version` number consistent.
+# This will affect the cmake command below.
+git config core.abbrev 9
+git fetch --unshallow || true
+git fetch --tags
+
+# Select xcode: latest version found on vmImage macOS-10.15 .
+DEVELOPER_DIR=/Applications/Xcode_12.4.app
+
+export ZIG_LOCAL_CACHE_DIR="$ZIGDIR/zig-cache"
+export ZIG_GLOBAL_CACHE_DIR="$ZIGDIR/zig-cache"
+
+# Build zig for host and use `Debug` type to make builds a little faster.
+
+cd $ZIGDIR
+mkdir build.host
+cd build.host
+cmake -G "Ninja" .. \
+  -DCMAKE_INSTALL_PREFIX="$(pwd)/release" \
+  -DCMAKE_PREFIX_PATH="$PREFIX_HOST" \
+  -DCMAKE_BUILD_TYPE="Debug" \
+  -DZIG_STATIC="OFF"
+
+# Build but do not install.
+ninja $JOBS
+
+ZIG_EXE="$ZIGDIR/build.host/zig"
+
+# Build zig for arm64 target.
+# - use `Release` type for published tarballs
+# - ad-hoc codesign with linker
+# - note: apple quarantine of downloads (eg. via safari) still apply
+
+cd $ZIGDIR
+mkdir build.arm64
+cd build.arm64
+cmake -G "Ninja" .. \
+  -DCMAKE_INSTALL_PREFIX="$(pwd)/release" \
+  -DCMAKE_PREFIX_PATH="$PREFIX_ARM64" \
+  -DCMAKE_BUILD_TYPE="Release" \
+  -DCMAKE_CROSSCOMPILING="True" \
+  -DCMAKE_SYSTEM_NAME="Darwin" \
+  -DCMAKE_C_FLAGS="-arch arm64" \
+  -DCMAKE_CXX_FLAGS="-arch arm64" \
+  -DCMAKE_EXE_LINKER_FLAGS="-lz -Xlinker -adhoc_codesign" \
+  -DZIG_USE_LLVM_CONFIG="OFF" \
+  -DZIG_EXECUTABLE="$ZIG_EXE" \
+  -DZIG_TARGET_TRIPLE="${ARCH}-macos" \
+  -DZIG_STATIC="OFF"
+
+ninja $JOBS install
+
+# Disable test because binary is foreign arch.
+#release/bin/zig build test
+
+# Show file size and dylibs.
+/bin/ls -lh release/bin
+/usr/bin/otool -L release/bin/zig
+/usr/bin/file release/bin/zig
```

Comment:
I gather this was to verify that `clang` successfully built an arm64 binary, am I correct? If so, is there any point in keeping these around? I don't imagine this ever regressing with each upcoming release of Apple build tools.

Reply:
removed and force-pushed.